src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -282,4 +280,49 @@ export default App.extend({
 
     this.showChildView('tooltip', tooltipView);
   },
+  showMultiEditButtonView() {
+    const multiEditButtonView = this.showChildView('filters', new MultiEditButtonView({
+      state: this.getState(),
+      selected: this.selected,
+      collection: this.collection,
+    }));
+
+    this.listenTo(multiEditButtonView, {
+      'click:cancel': this.onCancelMultiSelect,
+      'click:select': this.onClickMultiSelect,
+      'click:edit': this.onClickMultiEdit,
+    });
+  },
+  showEditActionsSidebar() {
+    const headerView = new BulkEditActionsHeaderView({
+      collection: this.selected,
+    });
+
+    const modal = Radio.request('modal', 'show:sidebar', {
+      headerView: headerView,
+    });
+
+    this.listenTo(headerView, {
+      'delete'(actions) {
+        this.onBulkDelete(actions);
+        modal.destroy();
+      },
+    });
+  },
+  showEditFlowsSidebar() {
+    const headerView = new BulkEditFlowsHeaderView({
+      collection: this.selected,
+    });
+
+    const modal = Radio.request('modal', 'show:sidebar', {
+      headerView: headerView,
+    });
+
+    this.listenTo(headerView, {
+      'delete'(flows) {
+        this.onBulkDelete(flows);
```

Comment:
I think if we want to make this an `on` Handler it should be `'delete': _.partial(this.onBulkDelete, modal)`  otherwise we can just call `flows.destroy();` `modal.destroy()` and clear the state here.  There should be no need to manually remove the models from `this.collection`